### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

File: cards/cardwithShadow.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    padding-bottom:0.3rem;
+
+}
+.card-btn:hover{
+    background-color: var(--card-primary-color);
+    margin-top:0.4rem;
+}
+
+.card-button{
+    text-align: center;
```
Comment (vanshitamoolchandani) [inline cards/cardwithShadow.css:105]: You can provide height to "card-button" <div> as its overlapping with price <div>

**Review by sri-rishi — CHANGES_REQUESTED** [review]
image
 
give some spacing between elements.

**Review by sri-rishi — CHANGES_REQUESTED** [review]
image
 
Give image border-radius so it looks same as div and make cards responsive

File: cards/cardwithoverlay.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+right:8px;
+bottom:45px;
+font-weight: 200;
+}
+.para-card .rating-card .rating-h{
+background-color: var(--card-badge-rating-bg);
+display: inline;
+padding:0.3rem;
+padding-left:0.5rem;
+padding-right:0.5rem;
```
Comment (vanshitamoolchandani) [inline cards/cardwithoverlay.css]: You can use shorthand property like padding: 0.3rem 0.5rem; 
It will give the same effect.

File: alerts.css [hunk 3]
```diff
@@ -49,38 +27,18 @@
...
->>>>>>> 34241759428d49fb2873f73b4e4ae05c401a9197
-    background-color: var(--greenalertCol);
-}
+    }
 
 .bluealert{
     background-color: var(--bluealertCol);
 }
 .yellowalert{
     background-color:var(--yellowalertCol) ;
```
Comment (sri-rishi) [inline alerts.css]: just a suggestion give color naming with " - " or use camel casing it's hard to read

File: cards/card1.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        </div>
+        <div class="para">
+            <p class="paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Sed blanditiis, 
+                consectetur officiis perspiciatis nostrum explicabo esse totam eius facere maiores repellat modi 
+                quibusdam cum velit aperiam in veritatis iusto consequuntur?</p>
+        </div>
+        <div class="card-button">
+            <button class="card-btn">show</button>
+            </div>
+    </div>
```
Comment (sri-rishi) [inline cards/card1.html]: remove comments

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by sri-rishi — APPROVED** [review]
changes good to merge

**Review by vanshitamoolchandani — APPROVED** [review]
Looks good to merge

**Pull request merged**